src/imeta.cpp

```diff
@@ -2190,8 +2193,8 @@ void usageMain() {
         " -h This help",
         " ",
         "Commands are:",
-        " add -d|C|R|u Name AttName AttValue [AttUnits] (Add new AVU triple)",
-        " adda -d|C|R|u Name AttName AttValue [AttUnits] (Add as administrator)",
+        " add  -d|C|R|u Name AttName AttValue [AttUnits] (Add new AVU triple)",
+        " adda -d|C|R|u Name AttName AttValue [AttUnits] [Deprecated] (Add as administrator)",
```

Comment:
or... we just remove it from the docs?   and line 2284 does the work to explain the -M option.

Reply:
sounds good

Reply:
so 4.3 will support it, we just won't document it is what you're saying?

Reply:
right, it will report that it is deprecated, suggest a better way, but will still execute.

Reply:
got it

Reply:
done